libs/domain/content/image/image.component.ts

```diff
@@ -39,7 +40,7 @@ export class ContentImageComponent extends ContentMixin<
   protected getStyles(): string | undefined {
     const { fit, position } = this.$options();
     let styles = '';
-    if (fit) styles += `--image-fit:${fit};`;
+    if (featureVersion < '1.4' && fit) styles += `--image-fit:${fit};`;
     if (position) styles += `--image-position:${position};`;
```

Comment:
What about this one? According the Pr's desc `position` should be specified via layouts as well